bot/cogs/reminders.py

```diff
@@ -56,12 +56,21 @@ def __init__(self, bot: Bot):
                 self.schedule_task(loop, reminder["id"], reminder)
 
     @staticmethod
-    async def _send_confirmation(ctx: Context, on_success: str) -> None:
+    async def _send_confirmation(
+        ctx: Context, on_success: str, reminder_id: str, delivery_dt: Optional[datetime]
```

Comment:
Prevailing style is for each parameter should be on a separate line. I notice the style above was already being used before your changes, but it could be a potential improvement here to make things more consistent with the rest of the code base.